lib/web/fetch/response.js

```diff
@@ -352,9 +352,7 @@ function cloneResponse (response) {
   // 3. If response’s body is non-null, then set newResponse’s body to the
   // result of cloning response’s body.
   if (response.body != null) {
-    newResponse.body = cloneBody(response.body)
-
-    streamRegistry.register(newResponse, new WeakRef(response.body.stream))
```

Comment:
I guess, it needed here to be?

`streamRegistry.register(newResponse, new WeakRef(newResponse.body.stream))`

Reply:
Yes, that's correct - I've updated the PR to drop all the reversion business and just make this one change + add a test